corehq/apps/userreports/tests/test_report_config.py

```diff
@@ -131,6 +131,7 @@ class ReportConfigurationDbTest(TestCase):
     @classmethod
     def setUpClass(cls):
         super(ReportConfigurationDbTest, cls).setUpClass()
+        delete_all_report_configs()
```

Comment:
interestingly, this doesn't look like it solved it. I wonder if `ReportConfiguration` is saving the two errored cases in `test_domain_is_required` and `test_config_id_is_required`

could try adding it to `tearDown`